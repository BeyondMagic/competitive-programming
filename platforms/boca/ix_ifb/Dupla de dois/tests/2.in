6
Daniel 30
Edson 40
Jeremias 50
Guilherme 65
Vinicius 70
Alberto 80
