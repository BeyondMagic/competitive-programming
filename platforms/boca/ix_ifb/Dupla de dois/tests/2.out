5
Daniel 30 Alberto 80
Edson 40 Vinicius 70
Jeremias 50 Guilherme 65
